charts/numaflow/values.yaml

```diff
@@ -0,0 +1,119 @@
+# Default values for numaflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+controller:
+  name: numaflow-controller
+  image:
+    repository: quay.io/numaproj/numaflow
+    tag: v1.0.0
+    pullPolicy: Always
+  # The number of controller replicas to run.
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1024Mi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  configMap:
+    # The name of the ConfigMap containing the controller configuration.
+    name: numaflow-controller-config
+    jetstream:
+      versions:
+        - version: latest
+          natsImage: nats:2.10.3
+        - version: 2.8.1
+          natsImage: nats:2.8.1
+        - version: 2.8.1-alpine
+          natsImage: nats:2.8.1-alpine
+        - version: 2.8.3
+          natsImage: nats:2.8.3
+        - version: 2.8.3-alpine
+          natsImage: nats:2.8.3-alpine
+        - version: 2.9.0
+          natsImage: nats:2.9.0
+        - version: 2.9.0-alpine
+          natsImage: nats:2.9.0-alpine
+        - version: 2.9.6
+          natsImage: nats:2.9.6
+        - version: 2.9.8
+          natsImage: nats:2.9.8
+        - version: 2.9.15
+          natsImage: nats:2.9.15
+        - version: 2.10.3
+          natsImage: nats:2.10.3
+  serviceAccount: numaflow-sa
+
+dexServer:
+  name: numaflow-dex-server
+  image:
+    repository: dexidp/dex
+    tag: v2.37.0
+    pullPolicy: Always
+  # The number of dex-server replicas to run.
+  replicaCount: 1
+  configMap: numaflow-dex-server-config
+  serviceAccount: numaflow-dex-server
+  secret:
+    name: numaflow-dex-secrets
+    data:
+      GITHUB_CLIENT_ID: ""
+      GITHUB_CLIENT_SECRET: ""
+
+server:
+  name: numaflow-server
+  image:
+    repository: quay.io/numaproj/numaflow
+    tag: v1.0.0
+    pullPolicy: Always
+  # The number of server replicas to run.
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1024Mi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  serviceAccount: numaflow-server-sa
+  volumeMounts:
+    - mountPath: /ui/build/runtime-env.js
+      name: env-volume
+      subPath: runtime-env.js
+    - mountPath: /ui/build/index.html
+      name: env-volume
+      subPath: index.html
+    - mountPath: /etc/numaflow
+      name: rbac-config
+  rbacConfigName: numaflow-server-rbac-config
+  cmdConfigName: numaflow-server-cmd-params-config
+  envsFromConfigMap:
```

Comment:
Instead of exposing all the items below, I think it would be better expose each of the them for the configmap template.